### DISCUSSION TITLE
The "online"/"offline" status message is kind of useless

### DISCUSSION THREAD

**Discussion by alstr** [tutao/tutanota#7320] [General]
* Often hangs at "Synchronizing: 94%"
* Sometimes it shows a reconnect button when offline, sometimes it doesn't, meaning you have to close/reopen the app until it eventually connects
* Clicking reconnect often does nothing - please at least give me an error message to tell me what has failed

**Comment by charlag** [comment]
I also have noticed the hang, we should investigate more.

It shows reconnect button if it's offline and isn't reconnecting. If you press a button and it doesn't reconnect then, well, reconnecting failed, there's not much more than that

**Reply by alstr** [reply]
It often doesn't show a reconnect button when it's offline. That doesn't really seem to make sense. I have to force close the app to get it to connect because there's no reconnect button to press.

Regarding the button, I just don't think it's good UX to simply fail with no acknowledgement. It just leaves the user in the dark, wondering if the button even does anything, especially if their internet connection is fine. If we implemented that logic everywhere, we wouldn't have error dialogs at all, and it would be infuriating.
